src/components/footer/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { translate } from 'react-i18next';
+import 'i18n/i18n';
+import styles from './footer.scss';
+
+const Footer = ({ t }) => (
+  <footer className={styles.footerMain}>
+    <div className={styles.footerInner}>
```

Comment:
Just wondering if we really need the extra div.... can we use only the `<footer>`?

Reply:
It allows creating margins for that line and keeping the overall content centered while the text is on the right (otherwise the whole line goes to the right or there's no margin). Also, I thought it wouldn't hurt if we decide to add more stuff or a background color to the footer later.

Reply:
that makes sense, thanks!